README.md

```diff
@@ -99,4 +99,25 @@ You can install the Fastlane dependencies by running `bundle install` from the r
 `which sentry-cli` is run. `which` does not exist on Windows, but the similarly named `where`
 command does, so you can get around any errors stemming from this by aliasing the command `where`
 to `which`.
- - You need to install [GitVersion](https://github.com/GitTools/GitVersion) yourself.
\ No newline at end of file
+ - You need to install [GitVersion](https://github.com/GitTools/GitVersion) yourself.
+
+## Release management
+
+1. After you've merged all PRs to be included in the release, ensure the `.update-priority`
+file is set correctly according to the table below. Use priority 2 as the default.
+
+| Update priority | Description | Examples | Update timeline for users |
+| --- | --- |--- | --- |
+| 0/1/2 | Very low  or low priority | UI touchups that don't impact functionality, releases with options to opt-in to beta features | No prompt initially. Optional prompt starting 14 days after release. Immediate update 21 days after release. |
+| 3 | Medium priority |  Medium-priority bug fixes, performance improvements, non-time-sensitive feature launches | No prompts for the first 3 days. Optional starting 4 days after release. Immediate update 21 days after release. |
```

Comment:
Does this number go any higher?

Reply:
Supposed to be 0-5, maybe I missed a few rows copying